charts/backingservices/charts/srs/README.md

```diff
@@ -15,23 +15,77 @@ The service deployment provisions runtime service pods along with a dependency o
 
 ### SRS Version compatibility matrix
 
-| Pega Infinity version | SRS version | Elasticsearch version | Description                                                                                                                                                                                                                                                                                                           |
-|-----------------------|-------------|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| < 8.6                 | NA          | NA                    | SRS can be used with Pega Infinity 8.6 and later                                                                                                                                                                                                                                                                      |
-| \>= 8.6              | 1.25.3  | 7.10.2, 7.16.3, and 7.17.9      | While SRS Docker images are certified against Elasticsearch versions 7.10.2, 7.16.3 and 7.17.9, Pega recommends using Elasticsearch version 7.17.9. To stay current with Pega releases, use the latest available SRS image 1.25.3.
+<table>
+    <thead>
+        <tr>
+            <th>Pega Infinity version</th>
+            <th>SRS version</th>
+            <th>K8's version</th>
+            <th>Authentication</th>
+            <th>Certified Elasticsearch version</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>< 8.6</td>
+            <td>NA</td>
+            <td>NA</td>
+            <td>NA</td>
+            <td>NA</td>
+            <td>SRS can be used with Pega Infinity 8.6 and later</td>
+        </tr>
+        <tr>
+            <td rowspan=4> >= 8.6 </td>
+            <td rowspan=4>1.26</td>
+            <td rowspan=2>< 1.25</td>
+            <td>No Authentication</td>
+            <td>7.10.2, 7.16.3 & 7.17.9</td>
+            <td>Pega recommends using Elasticsearch version 7.17.9.</td>
+        </tr>
+        <tr>
+            <td>Basic Authentication</td>
```

Comment:
Basic auth is supported by all ES versions, do we need to explicitly callout ?

Reply:
Hi @anilkumargedela, ES server 7.10.2 is not supported if the K8's version is >=1.25